src/dialog-renderer.ts

```diff
@@ -128,7 +144,7 @@ export class DialogRenderer implements Renderer {
   private setupClickHandling(dialogController: DialogController): void {
     this.stopPropagation = e => { e._aureliaDialogHostClicked = true; };
     this.closeDialogClick = e => {
-      if (!dialogController.settings.lock && !e._aureliaDialogHostClicked) {
+      if (dialogController.settings.backdropDismiss && !e._aureliaDialogHostClicked) {
```

Comment:
For the setting name we discussed `dismissableOverlay`, I'm good with any name might be worth polling in the aurelia channel to the core team about the name since this should be RC-worthy